docs/core/versions/index.md

```diff
@@ -2,7 +2,7 @@
 title: .NET Core Versioning
 description: .NET Core Versioning
 keywords: .NET, .NET Core
-author: richlander
+author: richlander, beleroy
```

Comment:
this line will probably give you a build error. it's just one author. since you significantly rewrote the topic, it's ok to change to your name. I'll go through the rest of the topic asap.